<launch>
  <node name="ttlidar_node"         pkg="ttlidar_node"  type="ttlidar_node" output="screen">
  <param name="serial_port"         type="string"       value="/dev/ttyUSB0"/>
  <param name="frame_id"            type="string"       value="laser"/>
  </node>
</launch>


